packages/contracts/test/test-utils/proxy.ts

```diff
@@ -0,0 +1,29 @@
+import {ContractFactory} from 'ethers';
+import {upgrades} from 'hardhat';
+
+type DeployOptions = {
+  constructurArgs?: unknown[];
```

Comment:
instead of `unkown[]` we could use generics to have more type safety

Reply:
Since OZ does use the same `unknown`, I went ahead with this. Should be enough for now.

Reply:
https://github.com/aragon/core/pull/154/commits/fffac2202953dbdf7bdd43d4c63d9f4f6e9daec8